doc/radosgw/adminops.rst

```diff
@@ -10,6 +10,54 @@ mechanism. Some operations require that the user holds special administrative ca
 The response entity type (XML or JSON) may be specified as the 'format' option in the
 request and defaults to JSON if not specified.
 
+Info
+====
+
+Get RGW cluser/endpoint information.
+
+:caps: info=read
+
+
+Syntax
+~~~~~~
+
+::
+
+	GET /{admin}/info?format=json HTTP/1.1
+	Host: {fqdn}
+
+
+Request Parameters
+~~~~~~~~~~~~~~~~~~
+
+None.
+
+
+Response Entities
+~~~~~~~~~~~~~~~~~
+
+If successful, the response contains the user information.
```

Comment:
```suggestion
If successful, the response contains the cluster unique identifier.
```

Reply:
?

Reply:
no, the idea here is we're returning an array of named elements--cluster_fsid is one of these

Reply:
Ahh, I see thanks, but still, is `user information` accurate? Thanks

Reply:
i.e., we may come up with other info elements at some point in future